podcasts/Onboarding/Import/ImportDetailsView.swift

```diff
@@ -24,33 +34,99 @@ struct ImportDetailsView: View {
 
                     appInstructions
 
+                    if app.id == .opmlFromURL {
+                        opmlImportView
+                    }
+
                     Spacer()
                 }.padding([.leading, .trailing], Constants.horizontalPadding)
             }
 
             // Hide button for other
-            if app.id != .other {
-                Button(app.id == .applePodcasts ? L10n.importInstructionsInstallShortcut : L10n.importInstructionsOpenIn(app.displayName)) {
-                    viewModel.openApp(app)
+            if !app.hideButton {
+                if app.id == .opmlFromURL {
+                   opmlViewButton
+                } else {
+                    Button(app.id == .applePodcasts ? L10n.importInstructionsInstallShortcut : L10n.importInstructionsOpenIn(app.displayName)) {
+                        viewModel.openApp(app)
+                    }
+                    .buttonStyle(RoundedButtonStyle(theme: theme))
+                    .padding([.leading, .trailing], Constants.horizontalPadding)
                 }
-                .buttonStyle(RoundedButtonStyle(theme: theme))
-                .padding([.leading, .trailing], Constants.horizontalPadding)
             }
         }.padding(.top, 16).padding(.bottom)
-        .background(AppTheme.color(for: .primaryUi01, theme: theme).ignoresSafeArea())
+            .background(AppTheme.color(for: .primaryUi01, theme: theme).ignoresSafeArea())
     }
 
     @ViewBuilder
     private var appInstructions: some View {
         let lines = app.steps.split(separator: "\n").map { String($0).trim() }
         VStack(alignment: .leading, spacing: 20) {
-           ForEach(lines, id: \.self) { line in
-               Text(line)
-                   .font(style: .subheadline, maxSizeCategory: .extraExtraExtraLarge)
-                   .foregroundColor(AppTheme.color(for: .primaryText01, theme: theme))
-                   .fixedSize(horizontal: false, vertical: true)
-           }
-       }
+            ForEach(lines, id: \.self) { line in
+                Text(line)
+                    .font(style: .subheadline, maxSizeCategory: .extraExtraExtraLarge)
+                    .foregroundColor(AppTheme.color(for: .primaryText01, theme: theme))
+                    .fixedSize(horizontal: false, vertical: true)
+            }
+        }
+    }
+
+    @ViewBuilder
+    private var opmlImportView: some View {
+        VStack {
+            TextField("https://...", text: $opmlURLText)
+                .textFieldStyle(RoundedBorderTextFieldStyle())
+                .autocapitalization(.none)
+
+            switch opmlImportResult {
+            case .none: Text("")
+            case .success:
+                Text(L10n.opmlImportSucceededTitle)
+                    .foregroundColor(Color.green)
+            case .failure:
+                Text(L10n.opmlImportFailedTitle)
+                    .foregroundColor(Color.red)
```

Comment:
Same as above, here we should probably use `support05`

Reply:
updated